code/game/objects/items/crayons.dm

```diff
@@ -136,7 +136,8 @@
 		busy = FALSE
 
 /obj/item/toy/crayon/attack(mob/M, mob/user)
-	var/huffable = istype(src,/obj/item/toy/crayon/spraycan)
+	if(istype(src, /obj/item/toy/crayon/spraycan))	// Eating the spraycan is TOO silly!
```

Comment:
since you're basically cutting out all the below behavior in the case of the spraycan, it might make more sense to instead override /obj/item/toy/crayon/spraycan/attack and put the early return in there

Reply:
Done.